actions/docs-verifier/src/RedirectionVerifier/RedirectionVerifier.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
It's weird that I didn't have this from the beginning 😄 
I think the package references below can be removed now.